### DISCUSSION TITLE
Cannot Parse HTTP Error Log with parse_apache_log

### DISCUSSION THREAD

**Discussion by tastyfrankfurt** [vectordotdev/vector#8429] [General]
Having issues with the parse_apache_log parser doesnt like our error logs, see below for example,
$ parse_apache_log!(s'[Fri Jul 23 00:28:19.615325 2021] [:error] [pid 17705] ipa: INFO: [jsonserver_session] fred@example.local: idview_show(None, version=u\\'2.237\\'): RequirementError', timestamp_format:"%a %b %d %H:%M:%S.%6f %Y", format: "error")
function call error for "parse_apache_log" at (0:246): failed parsing common log line

Thanks

Tasty

**Comment by jszwedko** [comment]
Hey @tastyfrankfurt . That seems like it could be a bug so I opened #8431 to track.

**Comment by bfung** [comment]
Fixed with references in :
* https://github.com/vectordotdev/vrl/issues/103
* https://github.com/vectordotdev/vector/pull/13581
